### PR TITLE
Add constants for GIT contact numbers and URLs

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Context

We reference a number of contact values that relate to Get into teaching, which we should store in one place.

## Changes proposed in this pull request

* Adds constants for GIT’s beta website URL and phone number, opening times and other GIT URLs we use
* Uses those constants throughout the application
* Removes guidance around right to work for the form that’s shown in support

## Guidance to review

This PR was reviewed once when it was in draft, but I’ve made a few more changes since, so worth another gander, I reckon.

## Link to Trello card

https://trello.com/c/wpgn9P98

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/docs/environment-variables.md#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by despo** [comment]
@paulrobertlloyd maybe consider putting these details in `config/locales/en.yml` as it makes them easier to find and update, and they shouldn't really be part of the configuration.

**Comment by malcolmbaig** [comment]
Good idea 👍🏽 

Agree with @despo on using i18n strings.

> There’s no way of using constants in Markdown files, or is there? I haven’t updated those, so these values are still not entirely centralised

We could potentially update [this helper](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/helpers/content_helper.rb) to process ERB, thereby allowing calls to `I18n` in the markdown, eg - https://github.com/DFE-Digital/apply-for-teacher-training/pull/3999

**Comment by paulrobertlloyd** [comment]
Thanks @despo, seems like localised strings is the way to go!

File: app/views/candidate_mailer/_get_help.text.erb
Comment (duncanjbrown) [inline app/views/candidate_mailer/_get_help.text.erb:5]: I think I'd have a single key for `get_into_teaching.online_chat_url`? If there's duplication across multiple GIT urls then it'll be adjacent in the YAML, and not knowing whether to including a slash when sticking paths on domains makes me anxious

**Comment by malcolmbaig** [comment]
https://github.com/DFE-Digital/apply-for-teacher-training/pull/3999 now merged, can use erb tags in .md files.

**Comment by paulrobertlloyd** [comment]
🎉

**Review by duncanjbrown — APPROVED** [review]
💯

**Comment by paulrobertlloyd** [comment]
Just going to review some of the URLs with @gregknight1, then this’ll be good to go - thanks all!

File: app/views/support_interface/application_forms/right_to_work_or_study/edit.html.erb
Comment (paulrobertlloyd) [inline app/views/support_interface/application_forms/right_to_work_or_study/edit.html.erb:28]: Note: have removed GIT content from this form as it’s seen by support agents and this guidance is intended for candidates. Leaving it here means it’ll probably fall out of sync with what’s shown in the candidate interface.

File: app/components/personal_statement_component.html.erb
Comment (paulrobertlloyd) [inline app/components/personal_statement_component.html.erb:19]: The content shown in the provider interface has fallen out of step with what’s shown to candidates. Not the first time this has happened, either. Surely there must be a better way to keep these in sync… maybe for starters we can add some comments in the code? @duncanjbrown

Comment (duncanjbrown) [reply app/components/personal_statement_component.html.erb:19]: This is a fact of life rn unfortunately, unless we keep track of all previous versions — applications submitted in the past ought to have the guidance from then, rather than now. We accepted this as a drawback when we did this in the first place, but we should def update this in the provider interface too now.

Comment (paulrobertlloyd) [reply app/components/personal_statement_component.html.erb:19]: Ah, the whole historical aspect. Gah!

> we should def update this in the provider interface too now

Just to be clear, that’s what this PR does.

Comment (paulrobertlloyd) [reply app/components/personal_statement_component.html.erb]: Having spoken to the ProVendor design team, we’re actually remove this guidance. So in this PR I’ll just limit the change to pointing to the new GIT constants. Much better.

File: config/locales/en.yml
Comment (duncanjbrown) [inline config/locales/en.yml:15]: Why `\00A0` instead of a space?

File: config/locales/en.yml
Comment (duncanjbrown) [inline config/locales/en.yml:22]: ✅ all URLs checked and correct

**Review by duncanjbrown — APPROVED** [review]
Approving (pending @Tomcowan1's content sign-off) with one last q!

File: config/locales/en.yml
Comment (paulrobertlloyd) [reply config/locales/en.yml:15]: Prevents an orphan appearing in the footer (which is coming in a follow-up PR).

So instead of this:

```
Monday to Friday, 8.30am to 5pm (except public
holidays)
```

we get this:

```
Monday to Friday, 8.30am to 5pm (except
public holidays)
```

Comment (duncanjbrown) [reply config/locales/en.yml:15]: Nice. Orphan! I haven't heard that expression since subediting class. Skills I never used, obviously

**Pull request merged**
